lib/order.rb

```diff
@@ -0,0 +1,22 @@
+class Order
+  attr_reader :my_order, :my_bill, :orde_details, :self_calculated_total, :total_amount
+  def initialize
+    @my_order = []
+    @my_bill = []
+    @self_calculated_total = 0
+  end
+
+  def add_items(dish, quantity, total)
```

Comment:
Good use of including the quantity as a second argument.

Total price though wouldn't be something the user would normally be asked to input when making an order for an individual dish. I think the instructions are telling us to ask the user to input the total at the very end ie if he pays the right amount, that doubles as a check that what they've input is the same as what the actual total cost is.